scripts/cli-test.sh

```
@@ -51,7 +51,7 @@ until [ "$($FM_BTC_CLIENT getreceivedbyaddress $PEG_OUT_ADDR 0)" == "0.00000500"
 done
 mine_blocks 10
 RECEIVED=$($FM_BTC_CLIENT getreceivedbyaddress $PEG_OUT_ADDR)
-[[ "$RECEIVED" = "0.00000500" ]]
+[[ "$RECEIVED" != "0.00000500" ]];
```

Why does this assert `!=` instead of the exact `=` amount?

---

Yeah, what happened here. :D

Also will fail silently which is not good.

---

We now know that we had a fiat CI again.